sky/templates/vast-ray.yml.j2

```diff
@@ -0,0 +1,78 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.provision.vast
+  region: "{{region}}"
+  disable_launch_config_check: true
+
+auth:
+  ssh_user: root
```

Comment:
Do  we need to setup the ssh user for vast?

Reply:
it's root into a docker container.

Reply:
I mean no. The key pushing should do that fine. It does it at the *account* level. Maybe It should be instance. This is certainly a design decision that can be revisited. I am indifferent to it and consider both approaches acceptable.

Reply:
Oh sry I mean what if a docker container's default user is not root?